### PR TITLE
Backend Image API

### LINKED ISSUE #288: Add Book Image Remove Feature

### PR DESCRIPTION
**Pull request by hkder**
## Feature Description/Implementation
Backend Image API (Closes #288 )

### CODE REVIEW

File: backend/hypothetical_books_backend/urls.py [hunk 1]
```diff
@@ -14,16 +14,16 @@
...
-    path(f'{api_prefix}buybacks', include('buybacks.urls')),
-]
+    path(f'{API_PREFIX}books', include('books.urls')),
+    path(f'{API_PREFIX}genres', include('genres.urls')),
+    path(f'{API_PREFIX}vendors', include('vendors.urls')),
+    path(f'{API_PREFIX}sales', include('sales.urls')),
+    path(f'{API_PREFIX}purchase_orders', include('purchase_orders.urls')),
+    path(f'{API_PREFIX}auth', include('authapp.urls', namespace='authentication')),
+    path(f'{API_PREFIX}buybacks', include('buybacks.urls')),
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```
Comment (danielfeinblatt) [inline backend/hypothetical_books_backend/urls.py:29]: What is being added to the `urlpatterns` list here and why?

Comment (hkder) [reply backend/hypothetical_books_backend/urls.py:29]: https://docs.djangoproject.com/en/4.1/howto/static-files/

Comment (hkder) [reply backend/hypothetical_books_backend/urls.py:29]: static(settings.STATIC_URL, document_root=settings.STATIC_ROOT) is `'^api/v1/static/(?P<path>.*)$'`

Comment (danielfeinblatt) [reply backend/hypothetical_books_backend/urls.py:29]: It seems to say that this only works in debug mode. On prod we aren't using debug mode. Will this be an issue?

Comment (hkder) [reply backend/hypothetical_books_backend/urls.py:29]: `--insecure` flag will be used with runserver so it is actually serving static files. In production we would serve these static files using Nginx but this is an easier path

Comment (danielfeinblatt) [reply backend/hypothetical_books_backend/urls.py:29]: Ok. As long as this isn't an issue within the code, then I'll approve

**Review by danielfeinblatt — APPROVED** [review]
LGTM

**Pull request merged**